content/en/security/logs.md

```diff
@@ -62,6 +44,17 @@ Additionally, certain features are not available at the moment to customers who
 
 If you have any questions about how the Log Management Service satisfies the applicable requirements under HIPAA, please contact your account manager.
 
+**Note:**
+
+HIPAA-enabled custommer used to require the use of specific endpoints to submit logs in order to enforce some specific encryptions. This is not needed anymore as that those encryptions are now enabled on all log submission endpoints.
+
+These are the legacy endpoints that are not supported after version {edit suggestion to add version}:
```

Comment:
Version?

Reply:
@NBParis Should we instead just phrase it as:

"These are legacy endpoints that may not be supported in the future."

Reply:
Yeah or we can even not say anything as I don't think we have a plan to deprecate them at some point. This might happen but we would likely do a DNS routing rather than removing them.

Reply:
ok - let's say "These legacy endpoints are still supported:"